package.json

```diff
@@ -7,12 +7,15 @@
     "dev:server": "convex dev --tail-logs",
     "dev:client": "next dev",
     "dev:init": "convex dev --once",
-    "start": "next start"
+    "start": "next start",
+    "testFunctionsExistingBackend": "just convex deploy && just convex env set IS_TEST true && jest",
+    "testFunctions": "node backendHarness.js 'npm run testFunctionsExistingBackend'"
   },
   "dependencies": {
     "@auth0/auth0-react": "^2.0.1",
     "chess.js": "^1.0.0-beta.3",
     "convex": "^1.10.0",
+    "convex-helpers": "file:../convex-helpers/packages/convex-helpers/convex-helpers-0.1.24.tgz",
```

Comment:
(I will fix this to point to a real `convex-helpers` version once we release and rebase)